### PR TITLE
Studio 1.2

### PR DESCRIPTION
**Pull request by h3xar0n**
Release documentation for Studio version 1.2.0 🙌

### CODE REVIEW

File: app/studio/changelog.md [hunk 1]
```diff
@@ -3,6 +3,25 @@ title: Kong Studio Changelog
...
 
+### 1.2.0
+**Released on 1/17/2020**
+
+#### Added
+
+* Added the ability to generate Declarative Configuration for Kong Gateway.
+* Added new Linter Tray in the Editor view that shows all validation and linting errors associated with your currently open document.
+* Added setting to disable clickable links in Explorer responses
+* Added ability to bulk-edit query parameters
```
Comment (nijikokun) [inline app/studio/changelog.md]: Add line:

```
- Proxy settings can now be applied to node requests.
```

File: app/_data/docs_nav_studio_1.2.x.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    - text: Linting in Kong Studio
+      url: /linting
+- title: Guides & References
+  items:
+    - text: API Design Best Practices
+      url: /best-practices
+    - text: Generating Kong Declarative Config with Kong Studio
+      url: /dec-conf-studio
+    - text: OpenAPI GraphQL Support
+      url: /graphql
```
Comment (h3xar0n) [inline app/_data/docs_nav_studio_1.2.x.yml]: @nijikokun I think this is causing a newline error

https://travis-ci.org/Kong/docs.konghq.com/builds/638583743#L369

```suggestion
      url: /graphql

```

File: app/studio/1.2.x/debugging-with-insomnia.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+Kong Studio leverages the powerful open source debugging tool [Insomnia](https://insomnia.rest).
+Once you've designed your spec, you can generate requests and fully test and debug your spec right inside Kong Studio.
+
+
+## Debugging with Insomnia
+
+1. Open the **Workspace** of the spec you wish to test.
+
+2. In the upper right hand corner click **Test**
```
Comment (h3xar0n) [inline app/studio/1.2.x/debugging-with-insomnia.md]:
```suggestion
2. In the upper right-hand corner, click **Test**.
```

File: app/studio/1.2.x/debugging-with-insomnia.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+![Test](https://doc-assets.konghq.com/studio/1.0/debugging/test.png)
+
+3. Kong Studio will prompt you to "Generate Requests", note that this will overwrite any existing test activity. Click **Generate Requests** to agree.
+
+![Generate Requests](https://doc-assets.konghq.com/studio/1.0/debugging/generate-requests.png)
+
+Kong Studio will generate the sample requests from the spec file and display them in the **Debug** view
```
Comment (h3xar0n) [inline app/studio/1.2.x/debugging-with-insomnia.md]:
```suggestion
Kong Studio will generate the sample requests from the spec file and display them in the **Debug** view.
```

File: app/studio/1.2.x/debugging-with-insomnia.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+![Sample Requests](https://doc-assets.konghq.com/studio/1.0/debugging/sample-requests.png)
+
+4. Click on a request on the **Workspace sidebar** to populate that request into the **Request Panel**
```
Comment (h3xar0n) [inline app/studio/1.2.x/debugging-with-insomnia.md]:
```suggestion
4. Click on a request on the **Workspace sidebar** to populate that request into the **Request Panel**.
```

File: app/studio/1.2.x/debugging-with-insomnia.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+![Request Panel](https://doc-assets.konghq.com/studio/1.0/debugging/request-panel.png)
+
+5. Click **Send** to test the request. The response will render in the **Response Panel**
```
Comment (h3xar0n) [inline app/studio/1.2.x/debugging-with-insomnia.md]:
```suggestion
5. Click **Send** to test the request. The response will render in the **Response Panel**.
```

File: app/studio/1.2.x/debugging-with-insomnia.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+![Response Panel](https://doc-assets.konghq.com/studio/1.0/debugging/response-panel.png)
+
+To lean more about generating requests and to full leverage the power of Insomnia, head over to [Insomnia's Documentation](https://support.insomnia.rest/)
```
Comment (h3xar0n) [inline app/studio/1.2.x/debugging-with-insomnia.md]:
```suggestion
To learn more about generating requests and to fully leverage the power of Insomnia, head over to [Insomnia's Documentation](https://support.insomnia.rest/).
```

File: app/studio/1.2.x/deploy-to-dev-portal.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+* Admin API privileges for the Workspace you will be connecting to
+* The Kong Developer Portal is enabled and running
+
+
+
+### Step 1. Connect to Kong Enterprise
+
+From Kong Studio:
+
+* Open the spec you want to deploy to the Developer Portal
```
Comment (h3xar0n) [inline app/studio/1.2.x/deploy-to-dev-portal.md]:
```suggestion
1. Open the spec you want to deploy to the Developer Portal
```

File: app/studio/1.2.x/deploy-to-dev-portal.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+* Click on the **caret menu** and select **Deploy to Kong Portal**
```
Comment (h3xar0n) [inline app/studio/1.2.x/deploy-to-dev-portal.md]:
```suggestion
1. Click on the **caret menu** and select **Deploy to Kong Portal**
```

File: app/studio/1.2.x/deploy-to-dev-portal.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+![Menu](https://doc-assets.konghq.com/studio/1.0/dev-portal/01-menu.png)
```
Comment (h3xar0n) [inline app/studio/1.2.x/deploy-to-dev-portal.md]:
```suggestion
    ![Menu](https://doc-assets.konghq.com/studio/1.0/dev-portal/01-menu.png)
```

File: app/studio/1.2.x/deploy-to-dev-portal.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+
+
+Kong Studio will prompt you for information to connect to your **Kong Admin API**
```
Comment (h3xar0n) [inline app/studio/1.2.x/deploy-to-dev-portal.md]:
```suggestion
    Kong Studio will prompt you for information to connect to your **Kong Admin API**
```

File: app/studio/1.2.x/deploy-to-dev-portal.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+
+
+* Enter your API URL
```
Comment (h3xar0n) [inline app/studio/1.2.x/deploy-to-dev-portal.md]:
```suggestion
1. Enter your API URL.
```

File: app/studio/1.2.x/deploy-to-dev-portal.md [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+    * >Note: The URL here is the **ADMIN** **API** url set by the `admin_api_uri` property
+* Enter the Workspace for the Developer Portal you want to upload the spec  to
```
Comment (h3xar0n) [inline app/studio/1.2.x/deploy-to-dev-portal.md]:
```suggestion
1. Enter the Workspace for the Developer Portal you want to upload the spec  to
```

File: app/studio/1.2.x/deploy-to-dev-portal.md [hunk 14]
```diff
@@ -0,0 +1,50 @@
...
+* If your Admin API uses RBAC, enter the Token for to access the Workspace
```
Comment (h3xar0n) [inline app/studio/1.2.x/deploy-to-dev-portal.md]:
```suggestion
1. If your Admin API uses RBAC, enter the Token for to access the Workspace
```

File: app/studio/1.2.x/deploy-to-dev-portal.md [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+
+
+![Connect to Kong](https://doc-assets.konghq.com/studio/1.0/dev-portal/02-connect-to-kong.png)
```
Comment (h3xar0n) [inline app/studio/1.2.x/deploy-to-dev-portal.md]:
```suggestion
    ![Connect to Kong](https://doc-assets.konghq.com/studio/1.0/dev-portal/02-connect-to-kong.png)
```

File: app/studio/1.2.x/deploy-to-dev-portal.md [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+
+* Click **Connect to Kong** to validate Kong Studio’s connection.
```
Comment (h3xar0n) [inline app/studio/1.2.x/deploy-to-dev-portal.md]:
```suggestion
1. Click **Connect to Kong** to validate Kong Studio’s connection. 
```

File: app/studio/1.2.x/deploy-to-dev-portal.md [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
+* Click **Connect to Kong** to validate Kong Studio’s connection. 
+
+
+### Step 2. Deploy a Spec file
+
+If successful, Kong Studio will then prompt you to specify a file name for the spec file - this will be the name of the file in the Developer Portal.
+
+![Deploy Spec](https://doc-assets.konghq.com/studio/1.0/dev-portal/03-deploy-spec.png)
+
+* Click **Upload to Kong Portal** to upload the spec file
```
Comment (h3xar0n) [inline app/studio/1.2.x/deploy-to-dev-portal.md]:
```suggestion
1. Click **Upload to Kong Portal** to upload the spec file
```

File: app/studio/1.2.x/deploy-to-dev-portal.md [hunk 18]
```diff
@@ -0,0 +1,50 @@
...
+*  If there is an existing file with the same name Kong Studio will prompt you to overwrite the existing file or cancel
```
Comment (h3xar0n) [inline app/studio/1.2.x/deploy-to-dev-portal.md]:
```suggestion
1. If there is an existing file with the same name Kong Studio will prompt you to overwrite the existing file or cancel
```

File: app/studio/1.2.x/editing-specs.md [hunk 19]
```diff
@@ -0,0 +1,89 @@
...
+In either case, importing a spec file will create a new **Workspace** with the same name as the spec file, and open the spec file in the Editor tab.  The **Workspace** is the *project* where you can edit and debug your spec. Each spec file you import will open into a new Workspace, or overwrite the current Workspace if you choose. 
+
+![Spec in Editor](https://doc-assets.konghq.com/studio/1.0/editing-specs/02-edit-spec.png)
+
+
+## Importing Specs from File
+
+1. Click on the **Workspace Dropdown** menu and select **Import/Export**
+
+![Import/Export Menu](https://doc-assets.konghq.com/studio/1.0/editing-specs/03-import-export.png)
```
Comment (h3xar0n) [inline app/studio/1.2.x/editing-specs.md]:
```suggestion
    ![Import/Export Menu](https://doc-assets.konghq.com/studio/1.0/editing-specs/03-import-export.png)
```

File: app/studio/1.2.x/deploy-to-dev-portal.md [hunk 20]
```diff
@@ -0,0 +1,50 @@
...
+1. Click on the **caret menu** and select **Deploy to Kong Portal**
+
+    ![Menu](https://doc-assets.konghq.com/studio/1.0/dev-portal/01-menu.png)
+
+
+    Kong Studio will prompt you for information to connect to your **Kong Admin API**
+
+
+1. Enter your API URL.
+    * >Note: The URL here is the **ADMIN** **API** url set by the `admin_api_uri` property
```
Comment (h3xar0n) [inline app/studio/1.2.x/deploy-to-dev-portal.md]:
```suggestion
    >Note: The URL here is the **ADMIN** **API** url set by the `admin_api_uri` property
```

**Pull request merged**